pkg/controller/atlasproject/private_endpoint.go

```diff
@@ -91,6 +91,7 @@ func getStatusForInterfaceConnections(ctx *workflow.Context, projectID string) w
 			return workflow.Terminate(workflow.Internal, err.Error())
 		}
 
+		// interfaceEndpoint.Status is for the interface endpoint AZURE
```

Comment:
```suggestion
		// interfaceEndpoint.Status is for the Azure and GCP interface endpoints
```

Until recently it was `AzureConnectionStatus` but they changed this to just Status with the added support of GCP.